docs: add description to change device assignments

manato opened the pull request:
This pull request aims to add steps to change the camera type assignment for user convenience.
I appreciate it if you check it. @yk-fujii

yk-fujii commented:
Thanks!

I would like to add to the table.

| overlay command | GMSL2 port No. | 1  | 2  | 3  | 4  | 5  | 6  | 7  | 8  |
|---------------------|----------------|----|----|----|----|----|----|----|----|
| TIERIV(Tier4) ISX021 GMSL2 Camera Device Tree Overlay | Camera type    | C1 | C1 | C1 | C1 | C1 | C1 | C1 | C1 |
| TIERIV(Tier4) IMX490 GMSL2 Camera Device Tree Overlay |Camera type    | C2 | C2 | C2 | C2 | C2 | C2 | C2 | C2 |
| TIERIV(Tier4) ISX021 IMX490 GMSL2 Camera Device Tree Overlay |Camera type    | C1 | C1 | C2 | C2 | C1 | C1 | C2 | C2 |

manato commented:
@yk-fujii
Thanks for your comment! I've modified the table and added some lines to describe the overlay command.
I'd appreciate it if you would check it.
